src/pages/getting-started/app-guides/app-authorization/index.mdx

```diff
@@ -7,12 +7,30 @@ tagline: "Find out how to implement an authorization flow and configure the App
 desc: "Find out how to implement an authorization flow and configure the App authorization building block."
 ---
 
+# Authorization for LiveChat apps
```

Comment:
ustaliłyśmy z Olą, że jesli będa większe zmiany w danym BB, to mam to rozdzielić na dwie osobne sekcje